build-linux.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Maybe use info from here [https://stackoverflow.com/a/1638397/7964431](https://stackoverflow.com/a/1638397/7964431) so we can do out of src build